sympy/series/sequences.py

```diff
@@ -291,6 +293,78 @@ def __getitem__(self, index):
             return [self.coeff(self._ith_point(i)) for i in
                     range(start, stop, index.step or 1)]
 
+    def find_linear_recurrence(self,n,d=None,gfvar=None):
+        """
+        Finds the shortest linear recurrence that satisfies the first n
+        terms of sequence of order `\leq` n/2 if possible.
+        If d is specified, find shortest linear recurrence of order
+        `\leq` min(d, n/2) if possible.
+        Returns list of coefficients or ``[]`` if no recurrence was found.
```

Comment:
The coefficients are returned as `[b(1), b(2), ...]` as defined in the recurrence relation: `a(n) = b(1)*a(n-1) + b(2)*a(n-2) + ... `This is the same as the coefficients list returned by Mathematica's `FindLinearRecurrence` function. I don't think there is a sympy function that generates the sequence from the coefficients list and initial conditions (similar to Mathematica's `LinearRecurrence` function.)